src/components/ScenarioParameters/components/ScenarioParametersInputs/components/TableExportDialog/TableExportDialog.js

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  FormControl,
+  Grid,
+  InputAdornment,
+  InputLabel,
+  MenuItem,
+  Select,
+  TextField,
+  Typography,
+} from '@mui/material';
+
+const ALLOWED_FILE_TYPES = [
+  {
+    extension: 'csv',
+    label: 'CSV',
+  },
+  {
+    extension: 'xlsx',
+    label: 'Excel',
+  },
+];
+
+export const TableExportDialog = ({ defaultFileName, labels, onClose, onExport, open }) => {
+  const defaultFileExtension = ALLOWED_FILE_TYPES[0].extension;
+
+  const [fileName, setFileName] = useState(defaultFileName);
+  const [fileType, setFileType] = useState(defaultFileExtension);
+
+  const selectFileName = (event) => {
+    setFileName(event.target.value);
+  };
+  const selectFileType = (event) => {
+    setFileType(event.target.value);
+  };
+  const confirmExport = () => {
+    const exportedFileName = `${fileName}.${fileType}`;
+    onExport(exportedFileName);
+  };
+
+  return (
+    <Dialog onClose={onClose} open={open} data-cy="table-export-dialog">
+      <DialogTitle>{labels.title}</DialogTitle>
+      <DialogContent sx={{ paddingBottom: '0' }}>
+        <Typography>{labels.label}</Typography>
```

Comment:
Maybe we can find a more explicit name for this element like _subtitle_ or something else ? :slightly_smiling_face:

Reply:
Good point, I renamed it to `exportDescription` to be extra unique :smile: (which is convenient when `grep`'ping through the code)